src/fmu/dataio/_export_item.py

```diff
@@ -48,6 +48,7 @@
     "velocity": None,
     "volumes": None,
     "volumetrics": None,  # or?
+    "kxh": None,  # KH-product
 }
```

Comment:
Perhaps `kxh` is easy to misunderstand? Are there better words? `permxheight`? `khproduct`?

Reply:
A challenge is confusion with horizontal permeability. E.g. "KH" will be misleading. My first association with `permxheight` is that it refers to `PERMX` (perm in the x-direction). But this is inconsistent, of course, since the same argument applies for `kxh`.

Reply:
Updated with `khproduct`.